### ISSUE TITLE
Remap Caps Lock to Ctrl but Caps Lock can still do Caps Lock

### ISSUE THREAD

**Issue by wuliuxiansheng** [blinksh/blink#365]
After I remap Caps Lock to Ctrl, I found that Caps Lock can function both as Ctrl and Caps Lock. Is it possible to separate these two keys, like map Ctrl to Caps Lock and map Caps Lock to Ctrl?

**Comment by jairajs89** [comment]
I have this issue as well. Particularly I launch `micro` to edit a file and the first few seconds caps functions like caps, and then all of a sudden the remap will start functioning properly (I have it mapped to ctrl)

**Comment by idvorkin** [comment]
I think this may be a regression as it used to work the last time I tried blink (months ago).

**Comment by carloscabanero** [comment]
Unfortunately I think this is just iOS11 doing its “magic”. We haven’t changed this at all from our side. Problem is the new iOS is slower to process when you switch, and even worst, if you go to a different app it might still be sending the same combo to Blink. It happens in other apps too, I have been able to type in notes or safari while not using it. Sometimes I even have to go to another app to be able to type on safari or notes.

We cannot map Ctrl to behave as Caps Lock though. 

Closing here, let me know if you have any other ideas to improve this :(
